screen/contributors/src/main/java/com/ivy/contributors/ContributorsViewModel.kt

```diff
@@ -49,22 +49,25 @@ class ContributorsViewModel @Inject constructor(
     }
 
     private suspend fun fetchContributors() {
-        val contributors = ivyWalletRepositoryDataSource.fetchContributors()?.map {
-            Contributor(
-                name = it.login,
-                photoUrl = it.avatarUrl,
-                contributionsCount = it.contributions.toString(),
-                githubProfileUrl = it.link
-            )
-        }
+        val contributorsResult = ivyWalletRepositoryDataSource.fetchContributors()
 
-        if (contributors != null) {
+        contributorsResult.fold({ errorMessage ->
```

Comment:
Add named arguments to the `fold` so it's readable. And follow the order of the arguments as defined in the fold function

Reply:
Ok